packages/augur-ui/src/modules/events/actions/listen-to-updates.test.js

```diff
@@ -0,0 +1,490 @@
+import mockStore from "test/mockStore";
+import { listenToUpdates } from "modules/events/actions/listen-to-updates";
+import * as augurjs from "services/augurjs";
+import * as loadMarketsInfoModule from "modules/markets/actions/load-markets-info";
+import * as loadUnclaimedFeesModule from "modules/markets/actions/market-creator-fees-management";
+import * as convertLogsToTransactionsModule from "modules/transactions/actions/convert-logs-to-transactions";
+import * as updateAssetsModule from "modules/auth/actions/update-assets";
+import * as loadReportingModule from "modules/reports/actions/load-reporting";
+
+jest.mock("services/augurjs");
+
+describe("events/actions/listen-to-updates", () => {
```

Comment:
this test fails. getting
![screen shot 2018-10-23 at 5 13 35 pm](https://user-images.githubusercontent.com/3970376/47394031-fff70100-d6e6-11e8-8162-1240fe959b58.png)